bootloader/ubootenv/env.go

```diff
@@ -75,21 +140,7 @@ func Create(fname string, size int) (*Env, error) {
 	return env, nil
 }
 
-// OpenFlags instructs open how to alter its behavior.
-type OpenFlags int
-
-const (
-	// OpenBestEffort instructs OpenWithFlags to skip malformed data without returning an error.
-	OpenBestEffort OpenFlags = 1 << iota
-)
-
-// Open opens a existing uboot env file
-func Open(fname string) (*Env, error) {
-	return OpenWithFlags(fname, OpenFlags(0))
-}
-
-// OpenWithFlags opens a existing uboot env file, passing additional flags.
-func OpenWithFlags(fname string, flags OpenFlags) (*Env, error) {
+func openNativeFormat(fname string, flags OpenFlags) (*nativeEnv, error) {
```

Comment:
any reason why it's createNative  but openNative*Format* ?

Reply:
no reason at all, I can change it